### PR TITLE
Cleanup/remove idea folder

### PR DESCRIPTION
**Pull request by Gounlaf**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,14 +1,16 @@
-.idea
```
Comment (blackandred) [inline .gitignore]: Shouldn't the `.idea` be there? I guess it should stay, so it would not be added anymore.

Comment (Gounlaf) [inline .gitignore]: No. Folder .idea is related to developer's tools, not to the application itself. Theses kind of rules must be in developer's global .gitignore. It's their responsibilities (also mine, yours) to not commit them.

If you decide to ignore .idea, you should (have to) ignore all kind of configurations folder related to IDEs... It make no sens :)

Comment (blackandred) [inline .gitignore]: I didn't know about the global .gitignore.
Ok then, thanks.

Comment (Gounlaf) [inline .gitignore]: https://github.com/github/gitignore

There are a lot of global rules and project specific rules :) (and there are some online tools based on this repository that let you mix all you need)

Comment (blackandred) [inline .gitignore]: Thanks.

**Pull request merged**